docs/submission/clinical-data-validation-rules.md

```diff
@@ -6,8 +6,52 @@ platform_key: DOCS_CLINICAL_VALIDATION_RULES
+- Treatment: `submitter_treatment_id`
+- Follow Up: `submitter_follow_up_id`
+
+These fields must to be coded specifically for ICGC-ARGO purposes using the following rules:
+
+- These identifiers should not be derived from IDs such as biobank or hospital identifiers or any other personal identifying information. These IDs are to be coded in such a way that they cannot be tracked back to the individual donors, except by the submitting program. Only the program will keep the key that will permit the data to be linked back to the individual donors. This key must not be communicated to the data users.
+- Identifiers are assigned by each submitting program and must be unique within all the data submitted by that program (no duplicate IDs allowed).
+- These identifiers should be values that uniquely identify the entity they refer to.
+- Identifiers referring to the same entity should be consistent across different submissions from the submitting program, and should not be re-used to refer to two different entities. For example, the same donor should not be assigned different identifiers in different files or subsequent data submissions.
+- Identifiers cannot begin with DO, SP, SA, PD, TR or FU. These prefixes are reserved for ICGC-ARGO IDs.
+
+### Primary Diagnosis, Treatment and Follow Up Identifiers:
+
+These identifier fields allow linking of the different clinical events and should be coded using the following rules:
+
+- Each primary diagnosis should be assigned a unique `submitter_primary_diagnosis_id`. In the case of multiple primary diagnoses, each primary diagnosis should be a different `submitter_primary_diagnosis_id`. You will be required to submit the `submitter_primary_diagnosis_id` in the `Specimen` file - this provides information about which primary diagnosis the specimen is linked to. The `submitter_primary_diagnosis_id` is also required in the `Treatment` table, so it is understood which primary diagnosis the treatment is being administered for.
+- Each treatment regimen in the `Treatment` table should be assigned a unique `submitter_treatment_id`. If the treatment regimen consists of chemotherapy, hormone therapy or radiation therapy, then you will use the same `submitter_treatment_id` in the appropriate `Chemotherapy`, `Radiation` or `Hormone Therapy` tables. For example, a treatment regimen consisting of Chemotherapy and Radiation therapy is assigned `cr01` as the `submitter_treatment_id` in the `Treatment` file. You would then submit the relevant clinical treatment information in the `Chemotherapy` and `Radiation` files using the same `submitter_treatment_id` (`cr01`) in the `Chemotherapy` and `Radiation` files. This allows the information in the two tables to be linked together so it is understood that the two therapies were combined.
+- Each follow up should be assigned a unique `submitter_follow_up_id`. Optionally, you can include the `submitter_treatment_id` the follow up may be linked to.
```

Comment:
suggestion: 
Each follow up should be assigned a unique `submitter_follow_up_id`. Optionally, if a follow up is linked to a specific treatment, you may include the `submitter_treatment_id` for that followup.